### DISCUSSION TITLE
Performance for each calendar month

### DISCUSSION THREAD

**Discussion by onstring** [ghostfolio/ghostfolio#984] [Ideas]
The overview page will show the portfolio performance. 

Wonder if we can add a historical performance with more details, for example, it will display a calendar like table for the past 12 months, and the performance in each month, both in % and absolute values gain/loss.

**Comment by ThowZzy** [comment]
Currently we can only select the "Year to date" but we cannot select a custom range. 
It would be super useful to be able to see the performance in previous years, year by year.

**Reply by dtslvr** [reply]
We have an open ticket that establishes the groundwork for the yearly performance feature (#5853). Would you like to work on that implementation, @ThowZzy?
